### PR TITLE
Remove hidden files from directory thumbnails

### PR DESCRIPTION
**Pull request by owencmiller**
This change prevents hidden files from appearing as directory thumbnails.

### CODE REVIEW

File: gallery/__init__.py [hunk 1]
```diff
@@ -476,6 +476,13 @@ def hide_file(file_id: int, auth_dict: Optional[Dict[str, Any]] = None):
         return "Permission denied", 403
 
     file_model.hidden = True
+
+    # Remove hidden file from directory thumbnails
+    for d in Directory.query.filter(Directory.thumbnail_uuid == file_model.thumbnail_uuid).all():
+        d.thumbnail_uuid = DEFAULT_THUMBNAIL_NAME
+
+    refresh_thumbnail()
```
Comment (mxmeinhold) [inline gallery/__init__.py]: @RamZallan do thumbnails get refreshed periodically or does it need to be explicitly done this way every time?

Comment (RamZallan) [reply gallery/__init__.py]: AFAIK we never refresh the thumbnail, though there is a CLI command exposed [here](https://github.com/ComputerScienceHouse/gallery/blob/f63e7e12ef8876d994fb6e7c756271f1ab749858/gallery/__init__.py#L311).

**Comment by RamZallan** [comment]
@owencmiller have you gotten a chance to test this locally?

File: gallery/__init__.py (hunk 1 above)
Comment (mxmeinhold) [reply gallery/__init__.py]: Then I guess this does need to be run every time, unless we want to set up some timed job to do this.

**Comment by owencmiller** [comment]
@RamZallan tested locally and working 👍

**Pull request merged**